astro/src/content/docs/apis/_user-response-body.mdx

```diff
@@ -190,7 +193,7 @@ import UserLoginFieldsResponse from 'src/content/docs/apis/_user-login-response.
     The time-step size in seconds. With the current implementation, this will always be 30.
   </APIField>
   <APIField name="user.twoFactor.methods[x].email" type="String">
-    The value of the email address for this method. Only present if <InlineField>user.twoFactor.methods[x].method</InlineField> is `email`.
+    The value of the email address for this method. Only present if <InlineField>user.twoFactor.methods``[x]``.method</InlineField> is `email`.
```

Comment:
I know this makes eslint happy, but I tried doing double ``[x]`` on my own branch and I don't think it looks right.